bifree.\_private\_tools.exceptions.BadCallError
===============================================

.. currentmodule:: bifree._private_tools.exceptions

.. autoexception:: BadCallError